src/grt/src/GlobalRouter.cpp

```diff
@@ -3427,8 +3411,17 @@ void GlobalRouter::reportLayerWireLengths()
 void GlobalRouter::reportNetWireLength(odb::dbNet* net,
                                        bool global_route,
                                        bool detailed_route,
-                                       bool verbose)
+                                       bool verbose,
+                                       const char* file_name)
 {
+  std::string file(file_name);
+  std::ofstream out;
+  if (!file.empty()) {
+    out.open(file, std::ios::app);
```

Comment:
Why app?  'trunc' seems a better fit.

Reply:
In that case you should remove the file at the start of the command to avoid appending to an already existing file from a previous run.